### DISCUSSION TITLE
[ showcase ] - An idea regarding documentation

### DISCUSSION THREAD

**Discussion by riccardodicurti** [10up/headstartwp#534] [Ideas]
First, a thank you. I started working with WordPress headless developing the frontend in plain React, then switched to next.js and recently I found this framework ( yes I tried the competition as well :) ). 

I am exploring the documentation and would like to ask about the possibility of adding a showcase. I get to talk to friends and colleagues, and it would be nice to be able to show some good completed projects in production. 

I saw that there are links to a demo project that is no longer live: 
https://headstartwp.10up.com/docs/learn/getting-started/quick-tutorial/#how-routing-works

**Comment by nicholasio** [comment]
Hey, can you drop the links to the sites you completed in production and we can see about "showcase from the community" section.

**Comment by riccardodicurti** [comment]
Here is the site: riccardodicurti.it

I should soon start a second one for tourist accommodation.
